x/ecocredit/core/msg_cancel_test.go

```diff
@@ -3,26 +3,25 @@ package core
 import (
 	"testing"
 
+	"github.com/regen-network/regen-ledger/types/testutil"
 	"github.com/stretchr/testify/require"
-
-	"github.com/cosmos/cosmos-sdk/testutil/testdata"
 )
 
 func TestMsgCancel(t *testing.T) {
 	t.Parallel()
 
-	_, _, addr1 := testdata.KeyTestPubAddr()
+	addr1 := testutil.GenAddress()
 
```

Comment:
would prefer if we could make these types of cleanups in separate PRs in the future, was a lot of non-related things to scroll through 😕

Reply:
I know, Initially I had it as a local function, but in review it was suggested to move it to testutils, so I moved forward and made the updates - it was already faster while waiting for the review.
In github, when you do review there is a "[ ] Viewed" button - with this you can quickly scan the file and click that button to fold the file. This is very efficient.